File: Servidor/InHomeTFGCarlosHernandez/src/main/java/org/tfg/inhometfgcarloshernandez/domain/model/Evento.java
package org.tfg.inhometfgcarloshernandez.domain.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalTime;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class Evento {
    private int id;
    private String tipo;
    private String nombre;
    private String votacion;
    private LocalTime horaComienzo;
    private LocalTime horaFin;
    private String organizador;
}

File: Cliente/InHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/inhometfgandroidcarloshernandez/data/model/response/Evento.kt
package com.example.inhometfgandroidcarloshernandez.data.model.response

data class Evento (
    val id : Int,
    val tipo: String,
    val nombre: String,
    val votacion: String,
    val horaComienzo: String,
    val horaFin: String,
    val organizador: String,
)